OptimizelySDK.Tests/EventTests/EventBuilderTest.cs

```diff
@@ -564,6 +564,7 @@ public void TestCreateConversionEventNoAttributesWithInvalidValue()
                                                     {"timestamp", timeStamp },
                                                     {"uuid", guid },
                                                     {"key", "purchase" },
+                                                    {"revenue", 42 },
                                                     {"tags",
```

Comment:
Updating TestCreateConversionEventNoAttributesWithInvalidValue is fine.
This will prove this test allows inclusion of {"revenue", 42} .  However, I think
we really ought to add another new regression test besides modifying this
old test.  The new regression unit test, let's call it TestGetRevenueValue perhaps,
should confirm these things:
*1 If no "revenue" key-value pair appears in eventTags dictionary input, then
GetRevenueValue returns null .
*2 If a "revenue" key-value pair appears in eventTags dictionary input and the
value is an integer or a string which can parse as an integer, then
GetRevenueValue returns the expected integer .
*3 If a "revenue" key-value pair appears in eventTags dictionary input and the
value is not an integer nor a string which can parse as an integer, then
GetRevenueValue logs a message and returns null .

Here is what objective-c-sdk does:
* Testing: OPTLYEventBuilderTest.m checkEventTags:withEvent
contains this line:
``    XCTAssert([revenue isKindOfClass:[NSNumber class]], @"revenue should be an NSNumber .");``
* OPTLYEventBuilder.m ``- (NSNumber *)revenueValue:(OPTLYProjectConfig *)config value:(NSObject *)value`` contains code logic which accomplishes all 3 bullets *1, *2, *3, above which I am proposing a new
csharp-sdk TestGetRevenueValue added to this P.R. should check.